model/ParsedInterface.java

```diff
@@ -3,13 +3,11 @@
 import java.io.File;
 import java.io.IOException;
 
-public interface ParserInterface {
+public interface ParsedInterface {
 
-  void setFile(File file);
+  File giveFile();
```

Comment:
@undsett how about getting rid of this guy also? Meaning, can we not have this method in the interface at all?

Reply:
Yes, we don't need it there. Just forgot to do this.